llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -91,7 +91,7 @@ using namespace Llpc;
 
 namespace SPIRV {
 
-cl::opt<bool> SPIRVGenFastMath("spirv-gen-fast-math", cl::init(true),
+cl::opt<unsigned> SPIRVGenFastMath("spirv-gen-fast-math", cl::init(0),
```

Comment:
SPIRVGenFastMath -> SPIRVGenFastMathFlags
spirv-gen-fast-math -> spirv-gen-fast-math-flags
It is no longer a boolean but instead a collection of flags